tests/pools/test_pool_rpc.py

```diff
@@ -793,8 +795,8 @@ async def tx_is_in_mempool(wid, tx: TransactionRecord):
                 fetched: Optional[TransactionRecord] = await client.get_transaction(wid, tx.name)
                 return fetched is not None and fetched.is_in_mempool()
 
-            await time_out_assert(5, tx_is_in_mempool, True, wallet_id, join_pool_tx)
-            await time_out_assert(5, tx_is_in_mempool, True, wallet_id_2, join_pool_tx_2)
+            await time_out_assert(10, tx_is_in_mempool, True, wallet_id, join_pool_tx)
+            await time_out_assert(10, tx_is_in_mempool, True, wallet_id_2, join_pool_tx_2)
```

Comment:
was this flaky? I would expect speeding up the tests would mean shorter timeouts

Reply:
I don't think they were flaky, but when running with -n 4 I saw this fail, so i increased it. I think creating plots was stalling the cores and therefore there were timeouts in the node.